test_settings.py

```diff
@@ -0,0 +1,41 @@
+"""
+These settings are here to use during tests, because django requires them.
+
+In a real-world use case, apps in this project are installed into other
+Django applications, so these settings will not be used.
+"""
+
+from os.path import abspath, dirname, join
+
+
+def root(*args):
+    """
+    Get the absolute path of the given path relative to the project root.
+    """
+    return join(abspath(dirname(__file__)), *args)
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'default.db',
+        'USER': '',
+        'PASSWORD': '',
+        'HOST': '',
+        'PORT': '',
+    }
+}
+
+INSTALLED_APPS = (
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'edx_rest_api_client',
+)
```

Comment:
Does this not need the middleware described in the README?

Reply:
Strictly speaking, it doesn't.  `RequestCacheMiddleware` is required by apps as a safety-net for clearing between requests.  The tests clear the `TieredCache` between tests.  Also, `TieredCacheMiddleware` is optional and provides additional features of `TieredCache` that aren't needed for tests.

Let me know if you think they should be added to avoid confusion, or if you think no one will ever consider this after this PR. :)

Reply:
Ok, just making sure the test setup is valid.